ballerina-shell/modules/shell-cli/src/main/java/io/ballerina/shell/cli/handlers/HelpCommand.java

```diff
@@ -47,13 +55,17 @@ public void run(String... args) {
             String helpFile = PropertiesLoader.getProperty(HELP_FILE);
             String helpContent = FileUtils.readResource(helpFile);
             ballerinaShell.outputInfo(helpContent);
+
```

Comment:
Unnecessary line

Reply:
Removed in a1286fe5ba1e38ae8718140c18a16fadd90542e6